validator/testcases/regex.py

```diff
@@ -426,6 +426,42 @@ def js_tests(self):
             compat_type='error')
 
 
+@register_generator
+class Gecko48RegexTests(CompatRegexTestHelper):
+    """Regex tests for Firefox 48 updates."""
+
+    VERSION = FX48_DEFINITION
+
+    def js_tests(self):
+        yield self.get_test(
+            r'\bProxy\.(create|createFunction)\b',
```

Comment:
I fixed this in 3dcebf8 to not use a regex which should produce much less false positives.